maestro-orchestra/src/main/java/maestro/orchestra/Orchestra.kt

```diff
@@ -401,16 +403,18 @@ class Orchestra(
                         return true
                     }
                     retryCenterCount++
-                }
-                else if (visibility >= command.visibilityPercentageNormalized) {
+                } else if (visibility >= command.visibilityPercentageNormalized) {
                     return true
                 }
             } catch (ignored: MaestroException.ElementNotFound) {
             }
             maestro.swipeFromCenter(direction, durationMs = command.scrollDuration)
         } while (System.currentTimeMillis() < endTime)
 
-        throw MaestroException.ElementNotFound("No visible element found: ${command.selector.description()}", maestro.viewHierarchy().root)
+        throw MaestroException.ElementNotFound(
+            "No visible element found: ${command.selector.description()}",
+            maestro.viewHierarchy().root
+        )
```

Comment:
I see no changes here is this because of some indentation changes?

Reply:
Intellij formatting